content/ui-patterns/loading.mdx

```diff
@@ -328,6 +328,25 @@ There are many cases where it will be obvious that a process has completed or fa
 
 For other cases, the status will need to be manually conveyed to assistive technologies. Use a `role="status"` message or put content in an `aria-live` element that communicates the status of a process.
 
+#### Filter results
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{alignItems: 'flex-start', gap: 3, mb: 3}}>
+
+<div>
+
+If the user needs to wait for filtering to complete, a screen reader should announce how many results were returned from filtering. For example: "No items match the filter", or "5 items match the filter".
+
+To ensure that assistive technology is informed about the filter update, put the message in an element with the following attributes: `aria-live="polite"` and `aria-atomic="true"`.
+
+Implementations of this pattern can be seen in the following examples:
+- [Lookbook - Autocomplete](https://view-components-storybook.eastus.cloudapp.azure.com/view-components/lookbook/inspect/primer/alpha/auto_complete/playground)
+- [Storybook - ActionList](https://primer.style/react/storybook/?path=/story/components-actionlist-examples--async-list-with-spinner)
```

Comment:
This example will still be wrong until https://github.com/primer/react/pull/3808 merges